websites/management/commands/backpopulate_groups.py

```diff
@@ -0,0 +1,36 @@
+""" Backpopulate website groups and permissions"""
+from django.core.management import BaseCommand
+
+from main.utils import now_in_utc
+from websites.models import Website
+from websites.permissions import create_global_groups, create_website_groups
+
+
+class Command(BaseCommand):
+    """ Backpopulate website groups and permissions """
+
+    help = "Backpopulate website groups and permissions"
+
+    def handle(self, *args, **options):
+
+        self.stdout.write(f"Creating website permission groups")
+        start = now_in_utc()
+
+        # Redo global grpups too in case permissions changed
+        created, updated = create_global_groups()
+        self.stdout.write(
+            f"Global groups: created {created} groups, updated {updated} groups"
+        )
+
+        for website in Website.objects.iterator():
+            created, updated, owner_updated = create_website_groups(website)
+            self.stdout.write(
+                f"{website.name} groups: created {created}, updated {updated}, owner updated: {str(owner_updated)}"
+            )
```

Comment:
If the entire set of OCW course sites has been imported into the database and this command is run, this output is going to be written that many times. That's a potentially huge number. I think it would be better to keep running totals then report them below in the last write.
```suggestion
        website_groups_created, website_groups_updated, owners_updated = 0, 0, 0  
        for website in Website.objects.iterator():
            created, updated, owner_updated = create_website_groups(website)
            website_groups_created += created
            website_groups_updated += updated
            owners_updated += 1 if owner_updated else 0
```

Reply:
If all that's reported is a list of created/updated owners and groups at then end, and it doesn't seem to match up with the number of websites, then you know something went wrong somewhere but no idea where.   So seeing even less value for this.  If something really goes wrong there should be an exception raised.  How about adding a `--verbose` option, so that if the numbers don't seem to match up at the end, you can run it again and see the counts per site to help narrow down where the problem might be occurring?